### PR TITLE
Expand the resources list

### PR DESCRIPTION
**Pull request by laurentlb**
- Bring back the Texture section, which was lost in a previous commit
- Add some compression tips
- Add link to Debris articles
- Add some notes about synths

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -99,18 +99,72 @@ <h5>At a glance</h5>
...
+            <a href="https://conspiracy.hu/articles/9/">GPU Based Texture Synthesis</a> discusses some advantages of using the GPU.
+            Generating textures on the CPU can be slow, so you'll probably want to use multiple threads.
+        </p>
+
+        <p>General introduction to texture generation:</p>
+        <ul>
+            <li><a href="http://www.upvector.com/?section=Tutorials&subsection=Intro%20to%20Procedural%20Textures">Intro to
+                    Procedural Textures</a>
+            <li><a href="https://www.ctrl-alt-test.fr/2018/texturing-in-a-64kb-intro/">Texturing in a 64kB intro</a>, an overview of
+                the texture generation in Immersion.
```
Comment (virtualzavie) [inline resources.html]:
```suggestion
            <li><a href="https://www.ctrl-alt-test.fr/2018/texturing-in-a-64kb-intro/">Texturing in a 64kB intro</a>, an overview of
                the texture generation in H – Immersion.
```
But removing the mention altogether works as well.

File: resources.html [hunk 2]
```diff
@@ -122,6 +176,7 @@ <h2>Making-ofs</h2>
             <li><a href="http://www.lofibucket.com/articles/64k_intro.html">How a 64k intro is made</a>, 2017
             <li><a href="https://geidav.wordpress.com/2013/04/14/making-of-turtles-all-the-way-down/">Making-of “Turtles all the way down”</a>, 2013
             <li><a href="http://peisik.untergrund.net/zine/gaia_machina/">Making of Gaia Machina</a>, 2012
+            <li><a href="https://fgiesen.wordpress.com/2012/02/13/debris-opening-the-box/">Debris: Opening the box</a>, 2012
             <li><a href="https://www.ctrl-alt-test.fr/2010/behind-incubation/">Behind Incubation</a>, 2010
         </ul>
```
Comment (virtualzavie) [inline resources.html]:
```suggestion
            <li><a href="https://www.ctrl-alt-test.fr/2010/behind-incubation/">Behind Incubation</a>, 2010
        </ul>

        <p>
        Not stricly a making of, <a href="https://fgiesen.wordpress.com/2012/02/13/debris-opening-the-box/">Debris: Opening the box</a> is a great series of highly technical articles from 2010 to 2012, about the design decisions behind the engine and the tools written for the mid 2000s Farbrausch intros.
        It covers topics such as <a href="https://fgiesen.wordpress.com/2010/03/28/how-to-generate-cellular-textures/">texture generation</a>, <a href="https://fgiesen.wordpress.com/2012/02/21/half-edge-based-mesh-representations-theory/">mesh generation</a> and <a href="https://fgiesen.wordpress.com/2011/01/24/x86-code-compression-in-kkrunchy/">compression</a>.
        </p>
 ```

**Comment by laurentlb** [comment]
applied the suggestions

**Review by virtualzavie — APPROVED** [review]
LGTM

**Pull request merged**